src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -113,9 +125,61 @@ case class Cluster(clusterName: ClusterName,
   def projectNameString: String = s"${googleProject.string}/${clusterName.string}"
 }
 
+object MachineConfig {
+  implicit val machineConfigSemigroup = new Semigroup[MachineConfig] {
+    def combine(defined: MachineConfig, default: MachineConfig): MachineConfig = {
+      val minimumDiskSize = 100
```

Comment:
I'm defining this here because if I do it through configs, it would mean I can't check for minimum diskSize in the Semigroup (which seemed simplest) - unless people have other ideas?